```
Traceback (most recent call last):

  File "/mnt/linux_data/Normaal/repos/openpype-for-normaal/openpype/hosts/blender/api/ops.py", line 145, in execute
    result = callback(*args, **kwargs)

  File "/mnt/linux_data/Normaal/repos/openpype-for-normaal/openpype/hosts/blender/api/ops.py", line 1048, in _build_first_workfile
    build_workfile()

  File "/mnt/linux_data/Normaal/repos/openpype-for-normaal/openpype/hosts/blender/scripts/build_workfile.py", line 564, in build_workfile
    build_anim(project_name, asset_name)

  File "/mnt/linux_data/Normaal/repos/openpype-for-normaal/openpype/hosts/blender/scripts/build_workfile.py", line 499, in build_anim
    load_subset(project_name, board_repre, "Background")

  File "/mnt/linux_data/Normaal/repos/openpype-for-normaal/openpype/hosts/blender/scripts/build_workfile.py", line 131, in load_subset
    loaders = loaders_from_representation(all_loaders, representation)

  File "/mnt/linux_data/Normaal/repos/openpype-for-normaal/openpype/pipeline/load/utils.py", line 771, in loaders_from_representation
    context = get_representation_context(representation)

  File "/mnt/linux_data/Normaal/repos/openpype-for-normaal/openpype/pipeline/load/utils.py", line 242, in get_representation_context
    assert representation is not None, "This is a bug"

AssertionError: This is a bug
```

I got this error when testing in SFTP.

--- a/openpype/hosts/blender/scripts/build_workfile.py
+++ b/openpype/hosts/blender/scripts/build_workfile.py
@@ -9,6 +9,7 @@ from openpype.client import (
     get_last_version_by_subset_id,
     get_representations,
 )
+from openpype.client.entity_links import get_linked_representation_id
 from openpype.hosts.blender.api.properties import OpenpypeContainer
 from openpype.lib.local_settings import get_local_site_id
 from openpype.modules import ModulesManager
@@ -68,17 +69,28 @@ def download_subset(
     if not representation:
         return
 
-    # Download representation on site
+    # Get sync server
     modules_manager = ModulesManager()
     sync_server = modules_manager.get("sync_server")
     local_site_id = get_local_site_id()
-    sync_server.add_site(
-        project_name,
-        representation["_id"],
-        local_site_id,
-        priority=99,
-        force=True,  # TODO meant to be removed when holding representation is fixed, see https://github.com/ynput/OpenPype/pull/4102
+
+    # Add linked representations
+    representation_ids = {representation["_id"]}
+    representation_ids.update(
+        get_linked_representation_id(
+            project_name, repre_id=representation["_id"]
+        )
     )
+
+    # Add local site to representations
+    for repre_id in representation_ids:
+        sync_server.add_site(
+            project_name,
+            repre_id,
+            local_site_id,
+            force=True,
+            priority=99,
+        )
 
     return representation
 

--- a/openpype/hosts/blender/scripts/build_workfile.py
+++ b/openpype/hosts/blender/scripts/build_workfile.py
@@ -88,7 +88,6 @@ def download_subset(
             project_name,
             repre_id,
             local_site_id,
-            force=True,
             priority=99,
         )
 

--- a/openpype/hosts/blender/scripts/build_workfile.py
+++ b/openpype/hosts/blender/scripts/build_workfile.py
@@ -77,6 +77,7 @@ def download_subset(
         representation["_id"],
         local_site_id,
         priority=99,
+        force=True,  # TODO meant to be removed when holding representation is fixed, see https://github.com/ynput/OpenPype/pull/4102
     )
 
     return representation
@@ -422,7 +423,9 @@ def build_anim(project_name, asset_name):
     """
     # Download not casting subsets
     layout_repre = download_subset(project_name, asset_name, "layoutMain")
-    board_repre = download_subset(project_name, asset_name, "BoardReference")
+    board_repre = download_subset(
+        project_name, asset_name, "BoardReference", "mov"
+    )
     camera_repre = download_subset(project_name, asset_name, "cameraMain")
     wait_for_download(project_name, [layout_repre, board_repre, camera_repre])
 

